src/ontology/ohd-edit.owl

```diff
@@ -5414,6 +5422,96 @@ AnnotationAssertion(dcterms:contributor obo:OHD_0005004 <https://orcid.org/0009-
 AnnotationAssertion(rdfs:label obo:OHD_0005004 "patient-reported unfavorable dental outcome"@en)
 EquivalentClasses(obo:OHD_0005004 ObjectIntersectionOf(obo:OHD_0000288 ObjectSomeValuesFrom(obo:IAO_0000136 obo:FMA_49184)))
 
+# Class: obo:OHD_0010000 (orofacial pain)
+
+AnnotationAssertion(obo:IAO_0000115 obo:OHD_0010000 "Pain originating in the orofacial region, which includes the mouth and face.")
```

Comment:
Rather than saying "Pain originating in X", I think it is better to say "Pain experience in X". Pain may originate from different region than where it is experienced.